rset.c

```diff
@@ -253,12 +258,23 @@ execute_remote(char *hostnames[], Label **route_labels, regex_t *label_reg) {
 					socket_path = NULL;
 					continue;
 				}
+
 				for (j=0; host_labels[j]; j++) {
 					if(regexec(label_reg, host_labels[j]->name, 1, &regmatch, 0) != 0)
 						continue;
 
 					log_msg(label_exec_begin_msg, hostname, host_labels[j]->name, 0);
 
+					/* restore */
+					if (restore_opt && host_labels[j]->export_paths[0])
```

Comment:
so, if a file does not exist, it just prints error and continues, right?

Reply:
correct, it just prints the error and continues